faqs/galaxy/datasets_BAM_dataset_identifiers.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Finding BAM dataset identifiers
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+Quickly learn what the identifiers are in any **BAM** dataset that is the result from mapping.
+
+Note: This will *not* work for "sequence-only" `bam` datasets, as these usually have no header.
```

Comment:
Number 4 is also a note, would it make sense to have them together at the end?

Reply:
Yes it does. Thanks for the comments.